ADQL.tex

```diff
@@ -746,7 +746,7 @@ \subsection{Mathematical and Trigonometrical Functions}
         \tabularnewline
 
         \hline
-        atan2(y,x) &
+        atan2(x,y) &
```

Comment:
Somebody else sanity check this for me, but I believe that the original text `atan2(y,x)` is correct (i.e. the conventional order) and the suggested change to `atan2(x,y)` is wrong.  See e.g. https://www.postgresql.org/docs/8.3/functions-math.html; also e.g.
```
select top 1 atan2(1,sqrt(3)) as pi_over_six from tap_schema.tables
```
from GAVO DC gives me PI/6 not PI/3.  The same order is used for atan2 in other languages as well (java, python).

Reply:
Thanks for picking that up. I have reverted the `atan2` change.